src/__experimental__/components/textarea/textarea.component.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from 'i18n-js';
+import { validProps } from '../../../utils/ether';
+import { InputPresentation } from '../input';
+import FormField from '../form-field';
+import CharacterCount from './character-count';
+import TextareaInput from './textarea-input.component';
+
+const i18nNumberOpts = { precision: 0 };
+
+class Textarea extends React.Component {
+  static propTypes = {
+    /** Character limit of the textarea */
+    characterLimit: PropTypes.string,
+    /** Type of the icon that will be rendered next to the input */
+    children: PropTypes.node,
+    /** The visible width of the text control, in average character widths */
+    cols: PropTypes.number,
+    /** Adds disabled property */
+    disabled: PropTypes.bool,
+    /** Stop the user typing over the characterLimit */
+    enforceCharacterLimit: PropTypes.bool,
+    /** Allows the Textareas Height to change based on user input */
+    expandable: PropTypes.bool,
+    label: PropTypes.string,
+    onChange: PropTypes.func,
+    placeholder: PropTypes.string,
+    /** Adds readOnly property */
+    readOnly: PropTypes.bool,
+    /** The number of visible text lines for the control */
+    rows: PropTypes.number,
+    /** One of type of size to apply to the textarea */
+    size: PropTypes.oneOf(['small', 'medium', 'large']),
```

Comment:
why not `size: PropTypes.oneOf[OptionsHelper.sizesRestricted]`?

Reply:
good point, changed, thanks